src/concrete/ml/torch/lora.py

```diff
@@ -184,14 +223,6 @@ def _replace(module: nn.Module):
 
         _replace(model)
 
-    def toggle_calibrate(self, enable: bool = True):
```

Comment:
isn't this method used in the Gpt2 notebook?

Reply:
Good catch. I will update GPT2 notebook. We should rerun the two notebook anyway.